src/CarbonAware.Tools/CarbonAware.Tools.WattTimeClient/src/WattTimeClient.cs

```diff
@@ -26,7 +26,7 @@ public class WattTimeClient : IWattTimeClient
         HttpStatusCode.Forbidden
     };
 
-    private HttpClient client;
+    public HttpClient client;
```

Comment:
does this need to be public?  I'm guessing this is for webapplicationfactory.  Can it be internal with an internalsVisibleTo attribute?  Can we hide it behind a property?

Reply:
Good catch, thank you.